CHANGELOG.md

```diff
@@ -1006,6 +1006,7 @@
 ## [release-102] 2022-03-23
 
 - Fix bug where Remember me was not working for users logging in with MFA
+- Add sector and sector category codes to radio button text when adding a new activity
```

Comment:
Needs moving to the `unreleased` section (rebase might keep trying to sneak it up).

Reply:
:+1: